keyboards/a_dux/keymaps/default/readme.md

```diff
@@ -0,0 +1,122 @@
+A usable default keymap for the Ferris keyboard
```

Comment:
Isn't the ferris?

Reply:
Ope, missed that! A lot of this config was sourced from the Ferris, including the default keymap. I'll get this fixed!